src/coreclr/minipal/dn-memmap.h

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#ifndef DN_MEMMAP_H
+#define DN_MEMMAP_H
+
+#include <stdint.h>
+
+class MemoryMappedFile
+{
+    size_t m_size;
+    void* m_address;
+    
+#ifdef TARGET_WINDOWS
+    HANDLE m_hFileMapping;
+#endif
+    
```

Comment:
```suggestion
```
This can be deleted from here (close the FileMapping handle at the end of the Open method instead)